python/pip_install/README.md

```diff
@@ -64,18 +64,21 @@ pip_install(
 #### Example `BUILD` file.
 
 ```python
-load("@py_deps//:requirements.bzl", "requirement")
+load("@py_deps//:requirements.bzl", "requirement", "whl_requirement")
 
 py_binary(
     name = "main",
     srcs = ["main.py"],
     deps = [
         requirement("boto3"),
     ],
+    data = [
+        whl_requirement("boto3"),
```

Comment:
nit: Think this needs a comment to communicate why you'd do this, as this particular target will run just fine without the addition, so I expect the example won't be all that clear to end-users.